initdb.py

```diff
@@ -114,8 +107,8 @@ def create_tables(cursor, connection):
             print("Creating table {}: ".format(name), end='')
             # Execute the CREATE xxx in TABLES
             cursor.execute(query)
-        except mysql.connector.Error as mysqlError:
-            raise Exception('There was a problem creating table {}:\n{}'.format(name, mysqlError))
+        except mysql.connector.Error as MySqlError:
```

Comment:
`MySqlError` violates [Google Python Style Guide: Naming](https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming)

make it something like: `mysql_error`